public class exceptionhandling {
    public static void main(String[] args) {
        try{
            int a= args.length;
            int a=2;
            System.out.println("a = "+a);
            int b=42/a;//Arithmatic Exception
            int c[]={10,15,20};
            c[22]=90; //Exception= Array Index out of Bounds
        } catch (Exception e) {
            //TODO: handle exception
            System.out.println("Array index oob: "+ e);
        } catch(Array{

        }

        
    }
}
